### PR TITLE
[KO] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
#893

### CODE REVIEW

File: ko/lessons/basics/control-structures.md [hunk 1]
```diff
@@ -174,3 +174,23 @@ with
...
+a = with {:ok, res} <- Map.fetch(m, :a),
+  true <- Integer.is_even(res) do
+    IO.puts "Divided by 2 it is #{div(res, 2)}"
+else
+  :error -> IO.puts "We don't have this item in map"
+  _ -> IO.puts "It's not odd"
+end
+```
+
+이는 오류 처리할 때 `case`같은 패턴매칭을 사용할 수 있도록 도와줍니다. 넘겨지는
```
Comment (devleoper) [inline ko/lessons/basics/control-structures.md]: Is there any intention you added a line break?

**Comment by devleoper** [comment]
Github's asking review feature doesn't work for those who don't have write access. So I'm manually asking folks to check out: @Dalgona @malkoG @riseshia

File: ko/lessons/basics/control-structures.md (hunk 1 above)
Comment (Dalgona) [inline ko/lessons/basics/control-structures.md]: - 이는 오류 처리할 때
+ 이는 오류를 처리할 때

File: ko/lessons/basics/control-structures.md [hunk 2]
```diff
@@ -174,3 +174,23 @@ with
...
+값는 첫 번째 매치하지 않은 표현식입니다.
```
Comment (Dalgona) [inline ko/lessons/basics/control-structures.md]:
```
- 넘겨지는 값는
+ 넘겨지는 값은
```

**Comment by Dalgona** [comment]
Anything else looks good! :rocket:

**Comment by Dalgona** [comment]
Oh, I just read the full text  and I found a minor spacing issue on line 9:

```
9- 오버헤드가 발생하지 않도록합니다.
9+ 오버헤드가 발생하지 않도록 합니다.
```

**Comment by Dalgona** [comment]
[`poolboy.md:139`](https://github.com/doomspork/elixir-school/blame/master/ko/lessons/libraries/poolboy.md#L139)

`- `여러 프로세스를 만드려고 해도
`+ `여러 프로세스를 만들려고 해도

Sorry for suggesting corrections one by one 😿

**Comment by marocchino** [comment]
Np at all. Happy Christmas.

**Comment by Dalgona** [comment]
Merry Christmas to all 🎄🎉

**Comment by malkoG** [comment]
LGTM :D

**Comment by marocchino** [comment]
@devleoper ping

**Comment by devleoper** [comment]
Pong! Sorry for making you wait.

**Comment by marocchino** [comment]
NP. xD

**Pull request merged**